identity-apps-core/apps/recovery-portal/src/main/webapp/includes/init-url.jsp

```diff
@@ -0,0 +1,163 @@
+<%--
+ ~
+ ~ Copyright (c) 2021, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ ~
+ ~ This software is the property of WSO2 LLC. and its suppliers, if any.
```

Comment:
Let's use the license for open source code here.

Reply:
fixed with https://github.com/wso2/identity-apps/pull/5817/commits/c4711e788c5f3c08e1e2906f20e2217b381f34e9